extensions/blocks/eventbrite/index.js

```diff
@@ -57,6 +63,7 @@ export const settings = {
 		// Modal button attributes, used for Button & Modal embed type.
 		text: {
 			type: 'string',
+			default: defaultModalButtonText,
```

Comment:
Based upon my other comment (https://github.com/Automattic/jetpack/pull/14585/files#r376028156), if we don't need to specify the `text` attribute in the example anymore, we also don't need to have a variable anymore:

```
text: {
  type: 'string',
  default: _x( 'Register', 'verb: e.g. register for an event.', 'jetpack' ),
},
```